package.json

```diff
@@ -54,7 +54,7 @@
     "jsonc-parser": "^2.0.2",
     "lint-staged": "^8.1.0",
     "monaco-editor": "^0.16.2",
-    "monaco-editor-core": "0.15.5",
+    "monaco-editor-core": "0.16.1",
```

Comment:
Why is the `peerDependencies` for `monaco-editor` is `0.19` but for `devDependencies` it is 0.16?

Reply:
I'm not sure. I think it was never really upgraded. I will create a separate PR to update all of them to 0.20 after we get this PR in.

Reply:
Also this reminds me, we need to add `js-yaml` to peer dependencies too. This is especially important when working with `monaco-editor-webpack-plugin` as per the suggestions in the issue https://github.com/pengx17/monaco-yaml/issues/20. Without that the final production Webpack build complains about the absence of `js-yaml`.